### PR TITLE
feat: schema review policy

### PR DESCRIPTION
**Pull request by candy2255**
PTAL

### CODE REVIEW

File: content/blog/schema-review-policy-mysql.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+title: "Introducing Schema Review Policy for MySQL"
+author: Candy
+published_at: 2022/06/15 10:39:19
+feature_image: /static/blog-changelog-assets/2022/06/schema-review-policy-cover.webp
+tags: Education
+featured: true
+description: This post introduces a new feature - schema review policy, which help DBAs and application developers compose better schema change statements.
+
+---
+
```
Comment (tianzhou) [inline content/blog/schema-review-policy-mysql.md:11]: For markdown, we just need to leave one empty line

File: content/blog/schema-review-policy-mysql.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+Bytebase is an open-source database DevOps tool, and it’s the GitLab for managing databases throughout the application development lifecycle. It offers a web-based workspace for DBAs and Developers to collaborate and manage the database change safely and efficiently.
+
+The trend of codifying database schema changes (aka Database as Code) has accelerated in recent years. Today, there exists a bunch of tools to check the code quality; however, the ability to check SQL statement quality is quite lacking.
+
+As the trailblazer to embrace Database as Code, Bytebase wants to lift the developer experience of working with SQL. Today we are introducing a new feature - schema review policy, a set of SQL quality checks to help DBAs and application developers compose better schema change statements.
+
+
+## What is Schema Review Policy?
+
+A schema review policy is a set of SQL-style rules to check the SQL statement quality. DBAs can configure schema review policies for each environment. Bytebase will check SQL statements automatically before executing schema changes. And block them if any configured policies are not met.
```
Comment (tianzhou) [inline content/blog/schema-review-policy-mysql.md:23]:
```suggestion
A schema review policy is a set of SQL-style rules to check the SQL statement quality. DBAs can configure schema review policies for each environment. Bytebase will check SQL statements automatically before executing schema changes, and block them if any configured policies are not met. 
```

File: content/blog/schema-review-policy-mysql.md (hunk 1 above)
Comment (candy2255) [reply content/blog/schema-review-policy-mysql.md:10]: Got it. Done.

File: content/blog/schema-review-policy-mysql.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+* Required columns: make sure all tables have the required columns
+* Disallow NULL: all columns can not have a NULL value
+
+### Engine
+
+* Require InnoDB
+
+InnoDB is the default storage engine of MySQL 5.5+ with powerful transaction features.
+
+
```
Comment (tianzhou) [inline content/blog/schema-review-policy-mysql.md:66]: just need 1 empty line. Please also check the rest of articles.

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Comment by boojack** [comment]
I just formated the content with https://codebeautify.org/markdown-formatter. FYI @candy2255

File: content/blog/schema-review-policy-mysql.md (hunk 3 above)
Comment (candy2255) [reply content/blog/schema-review-policy-mysql.md]: Done.

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**